.vale/styles/openly/Spelling.yml

```diff
@@ -0,0 +1,15 @@
+---
+extends: spelling
+message: "Did you really mean '%s'?"
+level: error
+code: false
+ignore:
+  - openly/tech-terms/security.txt
+  - openly/tech-terms/blockchain.txt
+  - openly/tech-terms/devops.txt
+  - openly/tech-terms/general.txt
+  - openly/tech-terms/techwriting.txt
+  - openly/tech-terms/mobile.txt
+  - openly/tech-terms/linux.txt
+  - openly/tech-terms/monitoring.txt
+  - openly/tech-terms/apis.txt
```

Comment:
Add blank line at end of file

Reply:
Marking resolved since this is now out of scope.